### ISSUE TITLE
viewer.html not working?! hyperlapse.tllabs.io outage?

### ISSUE THREAD

**Issue by MrJuldun38** [TeehanLax/Hyperlapse.js#33]
Hello,

I found this project and I tried to make a video using the files here on github but the file viewer.html seems to have a serious issue... It keeps displaying the same message: "ERROR: Could not retrieve panorama for the following reason: ZERO_RESULTS" an then a black screen appears...
Elsewhere, I can't access the hyperlapse.tllabs.io website. Is the project over?
Thank you for your time and support,
Julien

Btw I'm a complete newbie at programming, excuse me!

**Comment by kagu42** [comment]
im in the same boat please help

**Comment by lucanos** [comment]
+1

**Comment by maksvzw** [comment]
Yep, Teehan+lax closed their doors last year. I got it working a few months ago but it seems dead now :(

**Comment by wswld** [comment]
Just for the reference:
1. http://teehanlax.com/
2. https://medium.com/the-prototype/the-best-300-000-teehan-lax-ever-spent-4f5b059b37e6#.cmcymve7k
